### ISSUE TITLE
Plugin doesn't seem to work with all caps tags

### ISSUE THREAD

**Issue by Novbert** [pjeby/tag-wrangler#6]
I'm using all caps tags for certain abbreviations but it seem to break the plugin's capability of handling nested tags. 
I've tested the following setups:

1) Fresh vault with only a single file in it. The file content is this:
```
#ABC/DEF/GHI
#ABC/DEF/JKL
#ABC/DEF/MNO
```
Now if I try to rename ABC or DEF (to ABC2 or DEF2), nothing happens - there are zero replacements made.

2) Same setup but the file content is now this:
```
#abc/DEF/GHI
#abc/DEF/JKL
#abc/DEF/MNO
```
Now I can rename 'abc' to 'abc2', but still can't rename DEF to DEF2

3)
Same setup, this file content
```
#ABC/def/GHI
#ABC/def/JKL
#ABC/def/MNO
```
Now again, I can't rename ABC or 'def'. 

4) Even more interestingly I can rename lowercase tags to uppercase ones, but that seems to be an irreversible operation. So if I start with this:
```
#abc/def/GHI
#abc/def/JKL
#abc/def/MNO
```
I can rename 'abc' to ABC, but after that, I can't rename it back to 'abc'

So long story short, higher level renaming only work if all the tags are lowercase. Is this intentional? Could this be fixed?

**Comment by pjeby** [comment]
It's fixed in 0.3.2.  Thanks for reporting this!
